client/src/game/star.js

```diff
@@ -289,9 +294,9 @@ class Star extends EventEmitter {
     let totalKnownGarrison = (this.data.garrison || 0) + this._getStarCarrierGarrison()
 
     if ((totalKnownGarrison > 0) || (this._getStarCarriers().length > 0) || this._hasUnknownShips()) {
-      this.text_name.y = this.data.location.y
+      this.text_name.y = 0
     } else {
-      this.text_name.y = this.data.location.y - (this.text_name.height / 2)
+      this.text_name.y = - (this.text_name.height / 2)
```

Comment:
The space after the `-` isn't going to cause an issue is it?

Reply:
i dont think so, but i removed it anyways